tic-tac-toe-in-react/03-make-tic-tac-toe-game-buttons-interactive.md

```diff
@@ -0,0 +1,133 @@
+# Notes
+
+<TimeStamp start="0:02" end="0:07">
+
+We are going to add buttons in each cell to make our cells interactive
+
+</TimeStamp>
+
+<TimeStamp start="0:35" end="0:58">
+
+The resulting code for out buttons is: 
+
+```jsx
+ <button
+        css={{
+          fontSize: '2.5rem',
+          width: '100%',
+          height: '100%',
+        }}
+        onClick={() => {})}
+        type="button"
+      >
+
+ ```
+</TimeStamp>
+
+<TimeStamp start="1:45" end="1:52">
+
+For more information about  lifting of state and state updaters in React review the following documentation [here](https://reactjs.org/docs/lifting-state-up.html#lifting-state-up)
+
+</TimeStamp>
+
+<TimeStamp start="2:55" end="3:05">
+
```

Comment:
I would try to break these long notes down into smaller chunks
![Screen Shot 2021-06-30 at 2 36 08 PM](https://user-images.githubusercontent.com/518406/124021040-ade6fd00-d9b0-11eb-81db-432fecf5ae1e.png)

The goals is for them to look good in this format

Reply:
I have removed the pieces of code at the end of each lesson, this course is very practical, hence there is not a lot of room for notes.